app/helpers/application_helper.rb

```diff
@@ -1,2 +1,8 @@
 module ApplicationHelper
-end
+  def full_title(page_title = "")
+    base_title = t "title"
+    if !page_title.empty?
```

Comment:
thêm 1 case nếu không có page_title nhé, và dùng toán tử 3 ngôi cho trường hợp này